logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/team/TeamScope.kt

```diff
@@ -47,9 +45,4 @@ class TeamScope internal constructor(
         )
 
     val isSelfATeamMember: IsSelfATeamMemberUseCase get() = IsSelfATeamMemberUseCaseImpl(selfTeamIdProvider)
-
-    val approveLegalHoldRequestUseCase: ApproveLegalHoldRequestUseCase get() = ApproveLegalHoldRequestUseCaseImpl(
```

Comment:
Moved to `UserSessionScope` where we have all other legal hold related use cases so that we have all of them in the same space